src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const discountFromThirdDay = 20;
+  const discountFromSeventhDay = 50;
```

Comment:
```suggestion
  const smallDiscount = 20;
  const bigDiscount = 50;
```
Не привязывай название переменной к числам

Reply:
@vkryvytskyy  done